testdata/scripts/runtime-related.txt

```diff
@@ -0,0 +1,57 @@
+# test that we used all necessary dependencies
+[!darwin] exec bash runtime-related-tested.sh
```

Comment:
I think you want `[linux]`, because most windows boxes won't ship with bash

Reply:
expanding on that a bit: perhaps `[linux] [exec:bash]` to skip linux boxes that don't have bash installed, like alpine linux.

you could also go with `[!darwin] [exec:bash]` documenting that Mac often ships with a bash that's too old; up to you.